scripts/check-signed-off-by.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+MAIN_BRANCH=master
+
+# ignores last commit because travis/gh creates a merge commit
+commits=$(git log --format=%h ${MAIN_BRANCH}..HEAD | tail -n +2)
+
+for sha in $commits; do
+  actual=$(git show -s --format=%B ${sha} | sed '/^$/d' | tail -n 1)
```

Comment:
The Sob is not required to be the last thing in the commit. In fact, I think this would've failed on a few of mine, because when pulling someone's patch, I put my Sob after theirs, so the author's was the second to the last, and mine was before that. Also, JIRA, and other footer lines could come after the Sob.

Reply:
I changed to check every line looking for it.